File: occupy-server/src/main/java/com/mcm/network/messages/ConfigData.java
package com.mcm.network.messages;

import java.util.LinkedList;
import java.util.List;

/**
 * Created by Mehrdad on 16/12/17.
 */
public class ConfigData {
    public String version;
    public MapConfigData mapConfig = new MapConfigData();
    public List<TowerConfigData> towers = new LinkedList<>();
    public List<UnitConfigData> units = new LinkedList<>();
}

File: occupy/Assets/Scripts/Network/Messages/ConfigData.cs
using System;
using System.Collections;
using System.Collections.Generic;

[Serializable]
public class ConfigData : BaseObjectData{
	public string version;
	public MapConfigData mapConfig;
	public List<TowerConfigData> towers = new List<TowerConfigData>();
	public List<UnitConfigData> units = new List<UnitConfigData>();
}
